swift-sdk/Internal/InAppManager.swift

```diff
@@ -241,9 +241,18 @@ class InAppManager : NSObject, IterableInAppManagerProtocolInternal {
         }
     }
     
-    // Implement this
-    private func handleIterableCustomAction(name: String) {
-        
+    private func handleIterableCustomAction(name: String, forMessage message: IterableInAppMessage) {
+        guard let iterableCustomActionName = IterableCustomActionName(rawValue: name) else {
+            return
+        }
+
+        switch iterableCustomActionName {
+        case .delete:
+            remove(message: message)
+            break
+        case .dismiss:
```

Comment:
@tapashmajumder Since the message is now persisted for the inbox, does this also handle the case so that we won't display the message again as in in-app?

Reply:
@davidtruong Yes, the message should be marked as processed.